src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/SymbolCompletionProviderTests.vb

```diff
@@ -6495,5 +6495,24 @@ End Class
             VerifyItemIsAbsent(text, "X")
         End Sub
 
+        <WorkItem(4900, "https://github.com/dotnet/roslyn/issues/4090")>
+        <Fact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Sub ColorColorInstanceSharedRegression()
```

Comment:
Looking at the tests, we have a lot of coverage of instance vs shared for members of the current type, but nothing for shared/instance through member access. Historically, it appears that the findStatic and findInstance locals appeared as we implemented NameSyntax completion.